polysh/rb_tree.py

```diff
@@ -40,20 +40,21 @@ def __init__(self, key = None, value = None, color = RED):
         self.value = value
         self.nonzero = 1
 
-    def __nonzero__(self):
-        return self.nonzero
+    def __bool__(self):
+        return bool(self.nonzero)
```

Comment:
It would be better to change this variable to a `bool`.

Reply:
Done